pages/common/samtools.md

```diff
@@ -0,0 +1,36 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Used for reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM input file to BAM stream and save to file:
+
+`samtools view -S -b {{input.sam}} > {{output.bam}}`
+
+- Take input from stdin (-) and print the SAM header and any reads overlapping a specific region to stdout:
+
+`{{other_command}} | samtools view -h - chromosome:start-end`
+
+- Sort file and save to BAM (the output format is automatically determined from the output file's extension):
+
+`samtools sort {{input}} -o {{output.bam}}`
+
+- Index a sorted BAM file (creates {{sorted_input.bai}}):
```

Comment:
ps. If you're curious, here is the full description of the command output:

> If an output filename is given, the index file will be written to _out.index_. Otherwise, for a CRAM file _aln.cram_, index file _aln.cram.**crai**_ will be created; for a BAM file _aln.bam_, either _aln.bam.**bai**_ or _aln.bam.**csi**_ will be created, depending on the index format selected.

..confused yet? ;) 90% of people will be using this command with `.bam` files though, so I think it's clearest to make the example explicitly for that, creating `.bam.bai` files as we now have.